### PR TITLE
Builder3D typo/grammar fixes

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

**Comment by wwmayer** [comment]
No need to replace the backslash of doxygen tags with the "at" character -- both forms are correct.

File: src/Base/Builder3D.cpp [hunk 1]
```diff
@@ -69,10 +69,10 @@ Builder3D::~Builder3D()
 
 /**
  * Starts the definition of point set with the given point size and color.
- * If posible make not to much startPoints() and endPoints() calls. Try to put all
- * points in one set. 
+ * If possible don't make not to many startPoints() and endPoints() calls.
```
Comment (wwmayer) [inline src/Base/Builder3D.cpp]: That's not correct either: don't make ... not ... to

Comment (luzpaz) [reply src/Base/Builder3D.cpp]: good catch. will fix and push updated commit.

**Comment by luzpaz** [comment]
Removed all the doxy stuff + revised issue based on https://github.com/FreeCAD/FreeCAD/pull/1133#pullrequestreview-80682211

**Comment by luzpaz** [comment]
I added [skip ci] so travis perpetually shows tests are still pending but they have been bypassed.

**Comment by wwmayer** [comment]
Merged.

**Pull request closed without merging**